plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -1384,10 +1390,10 @@ public void testTimestampWithTimeZone(boolean insertWithTrino)
             tests.addRoundTrip(dataType, afterEpoch.atZone(kathmandu));
             tests.addRoundTrip(dataType, afterEpoch.atZone(fixedOffsetEast));
             tests.addRoundTrip(dataType, afterEpoch.atZone(fixedOffsetWest));
-            tests.addRoundTrip(dataType, afterEpoch.atZone(ZoneId.of("GMT")));
-            tests.addRoundTrip(dataType, afterEpoch.atZone(ZoneId.of("UTC")));
```

Comment:
`ZoneId.of("UTC")` occurs a number of times across the code base (and, `ZoneId.of("UTC")` is not the same thing as `ZoneOffset.UTC` and cannot be used interchangeably). Is it also being flagged by ZoneIdOfZ check, or this is an unrelated change?

Reply:
This is unrelated, in that I wanted to make it possible to attach the `@SuppressWarnings` annotation on something smaller than the entire method, so I refactored it slightly in spirit of other pieces of this class. Yes, I noticed that some of this is duplicated, but it's in `static` methods, which don't have access to instance fields (all the other fields are also non-`static` so I made these new ones to match for consistency).

Reply:
I backed out these changes, as I think now there's no point in additionally testing the "Z" zone, as the test is already including `UTC`.

Reply:
if ZoneId.of("Z") is the same as ZoneOffset.UTC, then it's not the same as ZoneId.of("UTC")

i remember some date/time formatting being different for ZoneId.of("UTC") and ZoneOffset.UTC

Reply:
Yes, and I don't claim that `ZoneId.of("UTC")` should be replaced. Anyway, this change is gone now, so...